src/main/content/_assets/js/blog.js

```diff
@@ -132,9 +192,9 @@ var blog = function(){
 
 
     function init() {
-        var tag = getTagFromUrl();
-        if(tag){
-            filterPosts(tag);
+        var tagList = getTagFromUrl();
+        if(tagList && tagList.length > 0){
```

Comment:
since `tagList` was initialized to `[]`, just checking `tagList.length > 0` should be good enough

Reply:
Got it, I'll take a look, thank you!